src/features/duplicates/components/ConfigureModal.tsx

```diff
@@ -11,47 +11,85 @@ import { FC, useState } from 'react';
 
 import FieldSettings from './FieldSettings';
 import messageIds from '../l10n/messageIds';
+import { PotentialDuplicate } from '../store';
+import PotentialDuplicatesLists from './PotentialDuplicatesLists';
 import useFieldSettings from '../hooks/useFieldSettings';
 import { useMessages } from 'core/i18n';
-import { ZetkinDuplicate } from '../store';
+import { ZetkinPerson } from 'utils/types/zetkin';
 
 interface ConfigureModalProps {
-  duplicate: ZetkinDuplicate;
+  potentialDuplicate: PotentialDuplicate;
   onClose: () => void;
   open: boolean;
 }
 
 const ConfigureModal: FC<ConfigureModalProps> = ({
-  duplicate,
+  potentialDuplicate,
   open,
   onClose,
 }) => {
   const fullScreen = useMediaQuery(theme.breakpoints.down('md'));
   const messages = useMessages(messageIds);
-  const { fieldValues, initialOverrides } = useFieldSettings(
-    duplicate.duplicatePersons
+  const [maxWidth, setMaxWidth] = useState<'sm' | 'lg'>('lg');
+
+  const potentialDuplicatesPersons: number[] =
+    potentialDuplicate?.duplicates.map((person) => person.id);
+
+  const [selectedIds, setSelectedIds] = useState<number[]>(
+    potentialDuplicatesPersons
```

Comment:
I would simplify this.

```suggestion
  const [selectedIds, setSelectedIds] = useState<number[]>(
    potentialDuplicate?.duplicates.map((person) => person.id) ?? [];
```